diff --git a/node_modules/easymde/src/js/easymde.js b/node_modules/easymde/src/js/easymde.js
index f7c1e46..5a3914f 100644
--- a/node_modules/easymde/src/js/easymde.js
+++ b/node_modules/easymde/src/js/easymde.js
@@ -12,7 +12,7 @@ require('codemirror/addon/search/searchcursor.js');
 require('codemirror/mode/gfm/gfm.js');
 require('codemirror/mode/xml/xml.js');
 var CodeMirrorSpellChecker = require('codemirror-spell-checker');
-var marked = require('marked/lib/marked');
+var marked = require('marked').marked;
 
 
 // Some variables
